app/src/main/java/org/simple/clinic/patient/PatientRepository.kt

```diff
@@ -57,20 +59,13 @@ class PatientRepository @Inject constructor(
   private var ongoingNewPatientEntry: OngoingNewPatientEntry = OngoingNewPatientEntry()
 
   fun search(criteria: PatientSearchCriteria): Observable<List<PatientSearchResult>> {
```

Comment:
Do you think now would be a good time to move search entirely out of `PatientRepository` as this does not need to be `SynceableRepository` or a repository at all?

Reply:
This change does not need to be a part of this PR.

Reply:
Good point, I will move the search capability out of the `PatientRepository` in another PR as discussed earlier.